pkg/snapshot/README.md

```diff
@@ -0,0 +1,225 @@
+# Snapshot (WIP)
+
+The snapshot package provides a CLI tool that allows dashboard users to export their 
+existing Tyk APIs and Security Policies into CRD YAML format that can be used by 
+Tyk Operator. 
+
+It can help you to migrate existing APIs and Policies to Kubernetes environment.
+
+> **Notes:** After the migration, please be reminded that you should stop using 
+the Dashboard UI to manage the migrated APIs and Policies. Please see 
+[Using Tyk Operator to enable GitOps with Tyk](https://tyk.io/docs/getting-started/key-concepts/gitops-with-tyk/) 
+for more information.
+
+This tool is provided as PoC only and bear some limitations and restrictions. 
+Please use with caution.
+
+---
+
+[Prerequisites](#prerequisites) | [Installation](#installation) | [Preparation](#preparation) | [Usage](#usage) | [Limitations](#limitations)
+
+---
+
+## Prerequisites
+
+1. Access to `tyk-operator` repository.
+2. [go](https://go.dev/doc/install)
+3. Credentials to connect Tyk Dashboard. Please visit [Tyk Docs](https://tyk.io/docs/tyk-stack/tyk-operator/installing-tyk-operator/#tyk-self-managed-hybrid) for details.
```

Comment:
```suggestion
3. Credentials to connect Tyk Dashboard/Gateway. Please visit [Tyk Docs](https://tyk.io/docs/tyk-stack/tyk-operator/installing-tyk-operator/#tyk-self-managed-hybrid) for details.
```
I believe it should work for both Pro and CE mode

Reply:
haven't tested for CE yet. I will test it and if it works, I will update the documentation accordingly 👍

Reply:
If it doesn't work, we should mention that in doc.

Reply:
It works with CE (for only ApiDefinitions). Updated documentation 👍